src/coreclr/src/gc/gcconfig.h

```diff
@@ -85,7 +85,7 @@ class GCConfigStringHolder
         "Specifies the name of the GC config log file")                                          \
     BOOL_CONFIG(GCNumaAware,   "GCNumaAware", true, "Enables numa allocations in the GC")        \
     BOOL_CONFIG(GCCpuGroup,    "GCCpuGroup", false, "Enables CPU groups in the GC")              \
-    BOOL_CONFIG(GCLargePages,  "GCLargePages", false, "Enables using Large Pages in the GC")     \
+    BOOL_CONFIG(GCLargePages,  "gcLargePages", false, "Enables using Large Pages in the GC")     \
```

Comment:
Why changing it to start with lower-case `gc` ?

Reply:
I was trying to follow what `gcConcurrent` does. The first 4 properties are inconsistent in the casing and I thought there is probably a convention there.

And then I discovered both `gcServer` and `gcConcurrent` have special paths in `GCToEEInterface::GetBooleanConfigValue`, leading me to think this might be the convention. 

Apparently, `GCRetainVM` doesn't follow the rule, with some more inspection, `gcConservative` and `gcForceCompact` doesn't follow the rule either.

SIgh, the casing is probably just random and historical. I will just change this back to what it was.